scripts/capture-base-provenance.py

```
@@ -87,7 +90,72 @@ def referrers_url(registry: str, repository: str, subject_digest: str) -> str:
+            "--show-error",
+            "--location",
+            "--connect-timeout",
+            "15",
+            "--max-time",
+            "60",
+        ]
+        if auth_cfg:
+            cmd += ["--config", auth_cfg]
+        cmd += [exchange_url]
+        try:
+            exchange = subprocess.run(cmd, check=True, capture_output=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)